guibot/match.py

```diff
@@ -28,7 +28,6 @@
 from .region import Region
 from .location import Location
 from .controller import Controller
-from .finder import Finder
```

Comment:
This should be recovered and the circular import must be handled on the side of newly annotated modules.

Reply:
Done